cmd/kubeadm/app/node/csr.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
2017

Reply:
Revert, this file was originally created in 2016 :wink: